### DISCUSSION TITLE
customising-aspnetcore-identity-without-editing-the-pagemodel/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#85] [Announcements]
# customising-aspnetcore-identity-without-editing-the-pagemodel/

In this post I show how you can replace the Razor views for the ASP.NET Core Identity default UI, without having to replace the code-behind PageModels.

https://andrewlock.net/customising-aspnetcore-identity-without-editing-the-pagemodel/

**Comment by Majestic-Mustang** [comment]
Great article Andrew.
Any chance you'd be able to write a blog article on how to style Identity razor pages to match with default Blazor theme (Blazor Server)?

I felt that it's so difficult to do that I went ahead and created an issue in aspnetcore Github repo?
https://github.com/dotnet/aspnetcore/issues/45174

Any help is appreciated.

[a comment by an automated account is not shown]

**Reply by andrewlock** [reply]
<em>[Sock](https://disqus.com/by/andrewlockdotnet/) (@andrewlock) commented [on Disqus](https://disqus.com/home/forum/andrewlock/) at June 02 2020, 01:51, in reply to [John Galt](https://disqus.com/by/jameshancock/)'s [comment](https://github.com/andrewlock/blog-comments/discussions/85#discussioncomment-6058575)</em>

---

<p>Yep, just omit the --files flag entirely. e.g.</p><p>dotnet aspnet-codegenerator identity -dc TestApp.Data.ApplicationDbContext</p><p>Check out the documentation here :) <a href="https://docs.microsoft.com/en-gb/aspnet/core/security/authentication/scaffold-identity" rel="nofollow noopener" title="https://docs.microsoft.com/en-gb/aspnet/core/security/authentication/scaffold-identity">https://docs.microsoft.com/en-gb/aspnet/core/security/authentication/scaffold-identity</a></p>

[a comment by an automated account is not shown]
